### PR TITLE
improve uuidv4 and update the deprecated library

### PR DESCRIPTION
**Pull request by rodriquelca**

### CODE REVIEW

File: dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue [hunk 1]
```diff
@@ -2,38 +2,21 @@
   <v-card elevation="0" class="fill-height" color="transparent">
     <v-scroll-y-transition mode="out-in">
       <v-row class="pa-2 fill-height" justify="space-between">
-        <v-col
-          style="min-width: 100px; max-width: 100%"
-          class="flex-grow-1 flex-shrink-0"
-          cols="5"
-        >
+        <v-col style="min-width: 100px; max-width: 100%" class="flex-grow-1 flex-shrink-0" cols="5">
```
Comment (fagubla) [inline dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue]: keep it in multiple lines, is most easy to read the code

File: dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue [hunk 2]
```diff
@@ -2,38 +2,21 @@
...
-          cols="7"
-          style="min-width: 100px"
-          class="flex-grow-0 flex-shrink-1 justify-center"
-        >
-          <div
-            tabindex="-1"
-            class="v-stepper__step subtitle-2 v-stepper__step--active"
-          >
+        <v-col cols="7" style="min-width: 100px" class="flex-grow-0 flex-shrink-1 justify-center">
+          <div tabindex="-1" class="v-stepper__step subtitle-2 v-stepper__step--active">
```
Comment (fagubla) [inline dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue]: keep it in multiple lines, is most easy to read the code

File: dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue [hunk 3]
```diff
@@ -2,38 +2,21 @@
...
-            <v-btn
-              color="primary"
-              plain
-              small
-              @click="selectTemplate"
-              class="vpm-main-panel-ds-btn float-end"
-              :disabled="schemaIsNotLoaded"
-            >
+            <v-btn color="primary" plain small @click="selectTemplate" class="vpm-main-panel-ds-btn float-end"
+              :disabled="schemaIsNotLoaded">
```
Comment (fagubla) [inline dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue]: keep it in multiple lines, is most easy to read the code

File: dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue [hunk 4]
```diff
@@ -114,4 +97,6 @@ const MainPanelDashboard = defineComponent({
 });
 export default MainPanelDashboard;
 </script>
-<style></style>
+<style>
```
Comment (fagubla) [inline dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue]: remove empty tag

File: dashboard/src/modules/Dashboard/Templates.vue [hunk 5]
```diff
@@ -1,9 +1,6 @@
 <template>
   <v-card elevation="0" class="fill-height" color="transparent">
-    <div
-      tabindex="-1"
-      class="v-stepper__step subtitle-2 v-stepper__step--active"
-    >
+    <div tabindex="-1" class="v-stepper__step subtitle-2 v-stepper__step--active">
```
Comment (fagubla) [inline dashboard/src/modules/Dashboard/Templates.vue]: keep it in multiple lines, is most easy to read the code

File: dashboard/src/modules/Dashboard/Templates.vue [hunk 6]
```diff
@@ -13,29 +10,16 @@
...
-      <v-card
-        flat
-        outlined
-        class="pa-2 ma-2 d-flex justify-center"
-        min-width="60"
-        min-height="80"
-        @click="createBlankForm"
-      >
+      <v-card flat outlined class="pa-2 ma-2 d-flex justify-center" min-width="60" min-height="80"
+        @click="createBlankForm">
```
Comment (fagubla) [inline dashboard/src/modules/Dashboard/Templates.vue]: keep it in multiple lines, is most easy to read the code

File: dashboard/src/modules/Dashboard/Templates.vue [hunk 7]
```diff
@@ -13,29 +10,16 @@
...
 
-      <v-card
-        flat
-        outlined
-        class="pa-2 ma-2 d-flex justify-center"
-        min-width="60"
-        min-height="80"
-        @click="false"
-      >
+      <v-card flat outlined class="pa-2 ma-2 d-flex justify-center" min-width="60" min-height="80" @click="false">
```
Comment (fagubla) [inline dashboard/src/modules/Dashboard/Templates.vue]: keep it in multiple lines, is most easy to read the code

File: dashboard/src/modules/Dashboard/Templates.vue [hunk 8]
```diff
@@ -44,31 +28,21 @@
       </v-card>
     </div>
 
-    <div
-      tabindex="-1"
-      class="v-stepper__step subtitle-2 v-stepper__step--active"
-    >
+    <div tabindex="-1" class="v-stepper__step subtitle-2 v-stepper__step--active">
```
Comment (fagubla) [inline dashboard/src/modules/Dashboard/Templates.vue]: keep it in multiple lines, is most easy to read the code

File: dashboard/src/modules/Dashboard/Templates.vue [hunk 9]
```diff
@@ -44,31 +28,21 @@
...
         <v-list-item-group color="primary">
-          <v-list-item
-            v-for="item in temples"
-            :key="item.title"
-            dense
-            @dblclick="createFromTemplate(item)"
-            @click="previewTemplate(item)"
-          >
+          <v-list-item v-for="item in temples" :key="item.title" dense @dblclick="createFromTemplate(item)"
+            @click="previewTemplate(item)">
```
Comment (fagubla) [inline dashboard/src/modules/Dashboard/Templates.vue]: keep it in multiple lines, is most easy to read the code

File: dunamis-editor/src/components/DynaformEditor/PalletePanel.vue [hunk 10]
```diff
@@ -1,49 +1,21 @@
 <template>
   <v-expansion-panels v-model="panel" accordion multiple>
-    <v-expansion-panel
-      class="caption"
-      v-for="(group, i) in paletteElements"
-      :key="i"
-    >
+    <v-expansion-panel class="caption" v-for="(group, i) in paletteElements" :key="i">
```
Comment (fagubla) [inline dunamis-editor/src/components/DynaformEditor/PalletePanel.vue]: keep it in multiple lines, is most easy to read the code

File: dunamis-editor/src/components/DynaformEditor/PalletePanel.vue [hunk 11]
```diff
@@ -1,49 +1,21 @@
...
-                  v-text="item.icon"
-                ></v-icon>
+          <draggable v-model="group.elements" :group="{
+            name: 'people',
+            pull: 'clone',
+            put: false,
+          }" :sort="false" class="flex-wrap" elevation="0" :clone="clone">
+            <div v-for="(item, n) in group.elements" :key="n" class="pa-1 vpm-drawer-item-list">
+              <v-card class="d-flex pa-1" flat :hover="true" color="transparent">
+                <v-icon small color="accent" class="pe-3 vpm-pallete-icon" v-text="item.icon"></v-icon>
```
Comment (fagubla) [inline dunamis-editor/src/components/DynaformEditor/PalletePanel.vue]: keep it in multiple lines, is most easy to read the code

File: dunamis-editor/src/components/DynaformEditor/PalletePanel.vue [hunk 12]
```diff
@@ -1,49 +1,21 @@
...
 
-                <v-list-item-title
-                  class="vpm-drawer-list-title caption"
-                  v-text="item.label"
-                ></v-list-item-title>
+                <v-list-item-title class="vpm-drawer-list-title caption" v-text="item.label"></v-list-item-title>
```
Comment (fagubla) [inline dunamis-editor/src/components/DynaformEditor/PalletePanel.vue]: keep it in multiple lines, is most easy to read the code

**Review by fagubla — CHANGES_REQUESTED** [review]

File: dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue (hunk 1 above)
Comment (rodriquelca) [reply dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue]: done

File: dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue (hunk 2 above)
Comment (rodriquelca) [reply dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue]: done

File: dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue (hunk 3 above)
Comment (rodriquelca) [reply dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue]: done

File: dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue (hunk 4 above)
Comment (rodriquelca) [reply dashboard/src/modules/Dashboard/MainPanelDashboardTemplates.vue]: done

File: dashboard/src/modules/Dashboard/Templates.vue (hunk 5 above)
Comment (rodriquelca) [reply dashboard/src/modules/Dashboard/Templates.vue]: done

File: dashboard/src/modules/Dashboard/Templates.vue (hunk 8 above)
Comment (rodriquelca) [reply dashboard/src/modules/Dashboard/Templates.vue]: done

File: dashboard/src/modules/Dashboard/Templates.vue (hunk 9 above)
Comment (rodriquelca) [reply dashboard/src/modules/Dashboard/Templates.vue]: done

File: dunamis-editor/src/components/DynaformEditor/PalletePanel.vue (hunk 10 above)
Comment (rodriquelca) [reply dunamis-editor/src/components/DynaformEditor/PalletePanel.vue]: done

File: dunamis-editor/src/components/DynaformEditor/PalletePanel.vue (hunk 11 above)
Comment (rodriquelca) [reply dunamis-editor/src/components/DynaformEditor/PalletePanel.vue]: done

File: dunamis-editor/src/components/DynaformEditor/PalletePanel.vue (hunk 12 above)
Comment (rodriquelca) [reply dunamis-editor/src/components/DynaformEditor/PalletePanel.vue]: done

**Review by fagubla — APPROVED** [review]

**Pull request merged**